### PR TITLE
Escape backslashes in copy to clipboard bibtex

### LINKED ISSUE #294: Bibtex copy to clipboard is missing backslashes for journal shortcuts
As reported by https://twitter.com/mpoessel/status/1084476039185793025, for example: "aj" instead of "\aj".

### PR DESCRIPTION
**Pull request by adrn**
This hopefully fixes #294.

This escapes backslashes in the string used by javascript to copy to clipboard.

### CODE REVIEW

File: acknowledging.html [hunk 1]
```diff
@@ -42,10 +42,10 @@
...
.... and {Kerzendorf}, W.~E. and {Khvalko}, A. and {King}, J. and {Kirkby}, D. and {Kulkarni}, A.~M. and {Kumar}, A. and {Lee}, A. and {Lenz}, D. and {Littlefair}, S.~P. and {Ma}, Z. and {Macleod}, D.~M. and {Mastropietro}, M. and {McCully}, C. and {Montagnac}, S. and {Morris}, B.~M. and {Mueller}, M. and {Mumford}, S.~J. and {Muna}, D. and {Murphy}, N.~A. and {Nelson}, S. and {Nguyen}, G.~H. and {Ninan}, J.~P. and {N{\\"o}the}, M. and {Ogaz}, S. and {Oh}, S. and {Parejko}, J.~K. and {Parley}, N. and {Pascual}, S. and {Patil}, R. and {Patil}, A.~A. and {Plunkett}, A.~L. and {Prochaska}, J.~X. and {Rastogi}, T. and {Reddy Janga}, V. and {Sabater}, J. and {Sakurikar}, P. and {Seifert}, M. and {Sherbert}, L.~E. and {Sherwood-Taylor}, H. and {Shih}, A.~Y. and {Sick}, J. and {Silbiger}, M.~T. and {Singanamalla}, S. and {Singer}, L.~P. and {Sladen}, P.~H. and {Sooley}, K.~A. and {Sornarajah}, S. and {Streicher}, O. and {Teuben}, P. and {Thomas}, S.~W. and {Tremblay}, G.~R. and {Turner}, J.~E.~H. and {Terr{\\'o}n}, V. and {van Kerkwijk}, M.~H. and {de la Vega}, A. and {Watkins}, L.~L. and {Weaver}, B.~A. and {Whitmore}, J.~B. and {Woillez}, J. and {Zabalza}, V. and {Contributors}, (Astropy},
```
Comment (bsipocz) [inline acknowledging.html]: the ő in my name should be `\H{o}` or `{\H o}`, this version will create an ó character.

Comment (adrn) [reply acknowledging.html]: That's weird - I just copy-pasted from ADS a few weeks ago!

Comment (adrn) [reply acknowledging.html]: In fact, yea, the ADS record is messed up - I'll fix it here, but you might want to send them a note: https://ui.adsabs.harvard.edu/#abs/2018AJ....156..123A/abstract

Comment (bsipocz) [reply acknowledging.html]: good catch, clearly they don't use the same bibtex to generate their listing...

Comment (bsipocz) [reply acknowledging.html]: Apparently it's an issue on the new platform and affects all my papers that I checkd. Using the old, reliable ADS, everything looks at it should: http://adsabs.harvard.edu/cgi-bin/nph-bib_query?bibcode=2018AJ....156..123A&data_type=BIBTEX&db_key=AST&nocookieset=1 😞

**Comment by bsipocz** [comment]
What about also adding the mention that this info is also available by `astropy.__citation__`?

**Comment by adrn** [comment]
No strong opinion, but I added a note

**Comment by eteq** [comment]
Thanks @adrn!

**Pull request merged**